bookshelf/src/main/webapp/base.jsp

```diff
@@ -34,9 +34,11 @@ Copyright 2015 Google Inc. All Rights Reserved.
         </div>
         <ul class="nav navbar-nav">
           <li><a href="/">Books</a></li>
-          <li><a href="/books/mine">My Books</a></li>
+          <div class="form-group ${isAuthConfigured ? '' : 'hidden'}">
```

Comment:
I'm not sure I understand.

>>  do all the property checking in one place, such as the init function in ListBookServlet

How would the requests to the CreateServlet get populated by the ListBookServlet code? I guess I could refactor some of this into a "populateConfiguration" method or something.